frontend/src/Judge/pages/Vote.js

```diff
@@ -40,6 +40,28 @@ const SubmissionContainer = styled.section`
 `
 
 class Vote extends Component {
+  handleKeyInput = e => {
+    const {
+      show,
+      handleNext,
+      handlePrevious,
+      submission,
+      previous,
+      next
+    } = this.props
+    if (e.key === 'ArrowRight') {
```

Comment:
I was going to comment this:

It looks like in IE and Edge have non-standard key event names for arrow keys.

See: 
- https://caniuse.com/#feat=keyboardevent-key
- https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/8860571/
- https://github.com/shvaikalesh/shim-keyboard-event-key

but then I remembered that React dispatches Synthetic Events, so we don't have to worry about that. (See: [React's source](https://github.com/facebook/react/blob/master/packages/react-dom/src/events/getEventKey.js))